src/Analyzer.Cli/CommandLineParser.cs

```diff
@@ -93,6 +100,31 @@ private Command SetupAnalyzeTemplateCommand()
             return analyzeTemplateCommand;
         }
 
+        private string GenerateResultString(Types.IEvaluation evaluation)
+        {
+            string resultString = "";
+
+            if (!evaluation.Passed)
+            {
+                if (evaluation.Results.Count() > 0)
+                {
+                    foreach (var result in evaluation.Results)
+                    {
+                        resultString += $"\n\tLine: {result.LineNumber}";
```

Comment:
`\n` by itself may not result in a proper new line in some Windows prompts or text viewers.  Use `Environment.NewLine` instead.

It would probably be helpful to create a constant in this file, something like `const string IndentedNewLine = Environment.NewLine + "\t";`